src/main/java/representer/RepresenterCli.java

```diff
@@ -47,23 +52,25 @@ public static void main(String[] args) {
 
         Representer representer = new Representer(modifierNormalizers, voidNormalizers);
         final String sourceFolder = contextPath + slug + "/" + JAVA_PROJECT_STRUCTURE;
+        logger.info("Search for source file into folder {}", sourceFolder);
         String[] sources = new File(sourceFolder).list();
+        if (sources == null) {
+            logger.error("Problems reading the folder");
+            System.exit(-1);
+        }
         Optional<String> javaSource =
                 Stream.of(sources).filter(s -> s.endsWith(".java")).findFirst();
         
         javaSource.ifPresent(s -> {
             try {
-
-                // String twoFer = "package myTest;\n\n\n\n class Twofer {"
-                // + " String twofer(String name) {"
-                // + " return \"One for \" + "
-                // + " (name != null ? name : \"you\") + \", one for me.\";"
-                // + " }" + " }";
-                String twoFer = new String(Files.readAllBytes(Paths.get(sourceFolder, "/", s)));
-                representer.generate(twoFer, new RepresentationSerializatorImpl(contextPath),
+                final String source = sourceFolder + "/" + s;
+                String sourceFileContent = new String(Files.readAllBytes(Paths.get(source)));
+                logger.info("Found source file {}", source);
+                representer.generate(sourceFileContent, new RepresentationSerializatorImpl(contextPath),
                         new MappingSerializatorImpl(contextPath));
+                logger.info("Generated representation file");
             } catch (IOException e) {
-                e.printStackTrace();
+                logger.error("Problems reading the source file", e);
```

Comment:
This line will only execute when the source files could not be found, right?

Reply:
This exception is thrown when there is some problem with the file: non-existent, wrong permission or something like this